pom.xml

```diff
@@ -90,6 +90,18 @@
         <!-- Old version until this issue is resolved: https://github.com/ImmobilienScout24/illegal-transitive-dependency-check/issues/30 -->
         <version>1.0.4</version>
       </dependency>
+      <dependency>
```

Comment:
Soft suggestion: the dependency management is usually part of the kie-parent too, so that you can just use the dependency and the version property is hidden for you.

Reply:
Merge of pr on `kie-parent` takes a lifetime to complete.

There is no problem for me to move these dependencies to parent, but can you, please, advise me how to get my pr reviewed&merged?

Reply:
Requested review from one of the owners of the repository there. Useful tip to avoid PR check failing when you have a dependency on change in other repository: create both PRs from branches that have the same name. This way, the PR check will build PRs from both repositories.

Reply:
That's really nice! Thank's for the tip. That PR is approved, yet still not merged...